src/qwed_new/phase4_runtime_probe.py

```
@@ -0,0 +1,56 @@
+import random
+import base64
+
+# --- Secrets: documented example values only ---
+AWS_ACCESS_KEY = "AKIAIOSFODNN7EXAMPLE"  # AWS docs example key
+STRIPE_KEY = "sk_live_4eC39HqLyjWDarjtT1zdp7dc"  # Stripe docs example key
+prod_api_secret_key = "xK9mQ2vL8nR4pW7yT3uI6oE1sA5dF8gH3jK"  # high-entropy generic
+JWT_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJVadQssw5c"
+
+
+def run_dynamic(user_expr):
+    return eval(user_expr)  # dynamic execution
```

Ruff (0.15.21)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)